nemo_skills/pipeline/generate.py

```diff
@@ -464,6 +448,7 @@ def generate(
         "If not specified, will use the default entrypoint for the server type.",
     ),
     dependent_jobs: int = typer.Option(0, help="Specify this to launch that number of dependent jobs"),
+    mount_paths: str = typer.Option(None, help="Comma separated list of paths to mount on the remote machine"),
```

Comment:
Please add some description of these new common parameters over here https://github.com/NVIDIA/NeMo-Skills/blob/main/docs/pipelines/index.md?plain=1#L76